### ISSUE TITLE
logging.basicConfig might be in the wrong place

### ISSUE THREAD

**Issue by frozflame** [belerico/py_asciimath#7]
I think it's not proper to config log level DEBUG inside many inner modules: 

```python
logging.basicConfig(format="%(levelname)s:%(message)s", level=logging.DEBUG)
```
It makes this package unusable in interactive mode, unless the user understand how to deal with the `logging` stuff. 

My suggestion is to put the `logging.basicConfig` only inside your entry point scripts / functions.

**Comment by belerico** [comment]
Hi, thanks for the suggestion :) If you want, you can also open a pull request ;)

**Comment by belerico** [comment]
Solved in 3d789f54a7696e747b09750dbd47e8af88e69e55
